spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-enhanced-ecommerce-tracker.spec.js

```diff
@@ -0,0 +1,295 @@
+/* eslint-env jasmine */
+
+describe('Google Analytics ecommerce tracking', function () {
```

Comment:
I'd consider merging the commit that adds this test with the commit that creates the code it refers to, since they're linked.

Reply:
Done.